compute_tools/src/pg_helpers.rs

```diff
@@ -244,6 +273,41 @@ pub fn get_existing_dbs(client: &mut Client) -> Result<HashMap<String, Database>
 
     Ok(dbs_map)
 }
+/// Build a list of existing Postgres databases
+pub async fn get_existing_dbs_async(
```

Comment:
Is it possible to remove the non-async versions of these functions?

Reply:
I'm saying can we change the code paths which are currently using the sync API for these functions to use the async API

Reply:
Nice job, thank you!